packages/core/collections/notes.js

```diff
@@ -347,11 +347,11 @@ export default class Notes extends Collection {
 
     for (const noteId of noteIds) {
       const note = this.note(noteId);
-      if (!note || note.deleted || !note.notebooks) {
-        continue;
-      }
 
       if (topicId) {
+        if (!note || note.deleted || !note.notebooks) {
```

Comment:
We should only check for `!note.notebooks` here. All other checks should remain where they were.

Reply:
okay